docs/workflows/README.md

```diff
@@ -68,20 +68,27 @@ A workflow is a provider-configured set of steps that describe the process to in
 
 AWS Step functions are described in detail in the AWS documentation but they provide several benefits which are applicable to AWS.
 
-* Prebuilt solution
-* Operations Visibility
-  * Visual diagram
-  * Every execution is recorded with both inputs and output for every step.
-* Composability
-  * Allow composing AWS Lambdas and code running in other steps. Code can be run in EC2 to interface with it or even on premise if desired.
-  * Step functions allow specifying when steps run in parallel or choices between steps based on data from the previous step.
-* Flexibility
-  * Step functions are designed to be easy to build new applications and reconfigure. We're exposing that flexibility directly to the provider.
-* Reliability and Error Handling
-  * Step functions allow configuration of retries and adding handling of error conditions.
-* Described via data
-  * This makes it easy to save the step function in configuration management solutions.
-  * We can build simple interfaces on top of the flexibility provided.
+- Prebuilt solution
+- Operations Visibility
+  - Visual diagram
+  - Every execution is recorded with both inputs and output for every step.
+- Composability
+  - Allow composing AWS Lambdas and code running in other steps. Code can be run in EC2 to interface with it or even on premise if desired.
+  - Step functions allow specifying when steps run in parallel or choices between steps based on data from the previous step.
+- Flexibility
+  - Step functions are designed to be easy to build new applications and reconfigure. We're exposing that flexibility directly to the provider.
+- Reliability and Error Handling
+  - Step functions allow configuration of retries and adding handling of error conditions.
+- Described via data
+  - This makes it easy to save the step function in configuration management solutions.
+  - We can build simple interfaces on top of the flexibility provided.
+
+#### AWS Step Functions considerations
```

Comment:
This is the actual change that I made, the rest is automatic linting changes.

Reply:
Good change, we should probably look into the difference between the vscode linter (I've done this on a few PRs) and the lint-md settings.